.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+  workflow_dispatch:
+
+jobs:
+  build_and_package:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-2019, macos-latest]
+
+    runs-on: ${{matrix.os}}
+    container: ${{ matrix.os == 'ubuntu-latest' && 'ghcr.io/f3d-app/f3d-ci' || null }}
+
+    env:
+      DISPLAY: :0
+
+    steps:
+
+    - name: Intall ninja Windows
+      if: matrix.os == 'windows-2019'
+      uses: seanmiddleditch/gha-setup-ninja@v3
+
+    - name: setup msvc Windows
+      if: matrix.os == 'windows-2019'
+      uses: TheMrMilchmann/setup-msvc-dev@v2
```

Comment:
I guess it's needed because you use ninja

Reply:
indeed